.travis.yml

```diff
@@ -25,7 +25,7 @@ matrix:
         - yarn run-examples tslint apollo-client
     - script:
         - yarn build
-        - yarn test
+        - 'yarn test && yarn coverage'
```

Comment:
Any reason not to put them on separate lines? I think the travis build will fail if you get a non-zero exit code from any of these. I guess maybe you could say they're semantically linked, in that running coverage without test doesn't make sense.

Also, why the quotes? I guess that's a yaml thing?

Reply:
> I think the travis build will fail if you get a non-zero exit code from any of these.
the build will fail but

https://docs.travis-ci.com/user/customizing-the-build/#customizing-the-build-step
> When one of the build commands returns a non-zero exit code, the Travis CI build runs the subsequent commands as well, and accumulates the build result.

and I don't want coverage to run on a failing build

> Also, why the quotes? I guess that's a yaml thing?

I wasn't sure what would happen with the `&`s. want me to remove them?

Reply:
Nah, it's fine, maybe a minor cleanup in the future, but iterating on travis stuff is sort of a pain anyway.